### DISCUSSION TITLE
29/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#194] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer no pude hacer nada de Android estuve muy engripada en cama.
Hoy hago el examen de emprendedurismo y me organizo con lo que hay q seguir.
El tiempo se pasa volando y me falta...

**Comment by fmdcba** [comment]
Hoy estuve repasando para emprendedurismo desde que llegué del laburo y recién terminé de rendir... Me quedé mareado con algunas preguntas, pero pude sacar una buena nota. Voy a seguir leyendo sobre lean startup y sumando con base en lo que tenemos anotado a modo hipotético.
